libs/components/code-examples/src/lib/modules/split-view/split-view/basic/example.component.html

```diff
@@ -1,77 +1,75 @@
-<div class="sky-padding-even-xl">
```

Comment:
Removing these wrappers so that the demos don't have excessive padding on the docs site.